roles/aws_setup_credentials/tasks/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - name: Define intial value for credentials
   ansible.builtin.set_fact:
-    aws_role_credentials: {}
+    aws_setup_credentials__output: {}
```

Comment:
Indeed this is to fix ansible-lint issues, there are other related changes. The changelog has been added
I don't know why the test passed 2 months ago, but if you tried to run them now on the `main` branch it would fail